src/server_manager/web_app/ui_components/outline-gcp-create-server-app.ts

```diff
@@ -298,25 +298,28 @@ export class GcpCreateServerApp extends LitElement {
     this.init();
     this.account = account;
 
-    this.billingAccounts = await this.account.listBillingAccounts();
-    const projects = await this.account.listProjects();
-    // TODO: We don't support multiple projects atm, but we will want to allow
-    //  the user to choose the appropriate one.
-    this.project = projects?.[0];
+    try {
+      this.billingAccounts = await this.account.listBillingAccounts();
```

Comment:
This is changing to `listOpenBillingAccounts` in the other PR

Reply:
Merged